wordpress-mobile/WordPress-iOS
# [Site Switcher] Older version search fix

Fixes #https://github.com/wordpress-mobile/WordPress-iOS/issues/23209

## Description
Fixes the the new site picker's search not working issue on versions prior to iOS 17.0

## Testing Steps
1. Install & Login Jetpack App on a device that is below iOS 17.0 (15.0 to 16.*)
2. Enable Site switcher Redesign Remote Feature Flag
3. Navigate to site switcher from My Site
4. ✅ Verify the search works fine.

## Regression Notes
1. Potential unintended areas of impact
N/A

5. What I did to test those areas of impact (or what existing automated tests I relied on)
N/A

6. What automated tests I added (or what prevented me from doing so)
N/A

PR submission checklist:


Testing checklist:
